core/templates/dev/head/pages/exploration-player-page/services/extract-image-filenames-from-state.service.spec.ts

```diff
@@ -17,128 +17,33 @@
+import { TestBed } from '@angular/core/testing';
 
-require('domain/exploration/ExplorationObjectFactory.ts');
-require(
-  'pages/exploration-player-page/services/' +
-  'extract-image-filenames-from-state.service.ts');
-require('services/context.service.ts');
+import { CamelCaseToHyphensPipe } from
+  'filters/string-utility-filters/camel-case-to-hyphens.pipe';
+import { ContextService } from 'services/context.service';
+import { ExplorationObjectFactory } from
+  'domain/exploration/ExplorationObjectFactory';
+import { ExtractImageFilenamesFromStateService } from
+  // eslint-disable-next-line max-len
+  'pages/exploration-player-page/services/extract-image-filenames-from-state.service';
 
-describe('Extracting Image file names in the state service', function() {
-  beforeEach(function() {
-    angular.mock.module('oppia');
-    angular.mock.module('oppia', function($provide) {
-      $provide.value(
-        'AnswerGroupObjectFactory', new AnswerGroupObjectFactory(
-          new OutcomeObjectFactory(new SubtitledHtmlObjectFactory()),
-          new RuleObjectFactory()));
-      $provide.value('FractionObjectFactory', new FractionObjectFactory());
-      $provide.value(
-        'HintObjectFactory', new HintObjectFactory(
-          new SubtitledHtmlObjectFactory()));
-      $provide.value(
-        'OutcomeObjectFactory', new OutcomeObjectFactory(
-          new SubtitledHtmlObjectFactory()));
-      $provide.value(
-        'ParamChangeObjectFactory', new ParamChangeObjectFactory());
-      $provide.value(
-        'ParamChangesObjectFactory', new ParamChangesObjectFactory(
-          new ParamChangeObjectFactory()));
-      $provide.value(
-        'ParamSpecObjectFactory',
-        new ParamSpecObjectFactory(new ParamTypeObjectFactory()));
-      $provide.value(
-        'ParamSpecsObjectFactory',
-        new ParamSpecsObjectFactory(
-          new ParamSpecObjectFactory(new ParamTypeObjectFactory())));
-      $provide.value('ParamTypeObjectFactory', new ParamTypeObjectFactory());
-      $provide.value(
-        'RecordedVoiceoversObjectFactory',
-        new RecordedVoiceoversObjectFactory(new VoiceoverObjectFactory()));
-      $provide.value('RuleObjectFactory', new RuleObjectFactory());
-      $provide.value(
-        'SubtitledHtmlObjectFactory', new SubtitledHtmlObjectFactory());
-      $provide.value('UnitsObjectFactory', new UnitsObjectFactory());
-      $provide.value('VoiceoverObjectFactory', new VoiceoverObjectFactory());
-      $provide.value(
-        'WrittenTranslationObjectFactory',
-        new WrittenTranslationObjectFactory());
-      $provide.value(
-        'WrittenTranslationsObjectFactory',
-        new WrittenTranslationsObjectFactory(
-          new WrittenTranslationObjectFactory()));
-    });
-    // Set a global value for INTERACTION_SPECS that will be used by all the
-    // descendant dependencies.
-    angular.mock.module(function($provide) {
-      $provide.constant('INTERACTION_SPECS', {
```

Comment:
This doesn't seem to be replicated. Was it not needed?

Reply:
No it's not needed after the migration.